seed/views/v3/data_quality.py

```diff
@@ -94,33 +94,45 @@ def _get_severity_from_js(severity):
 
 
 class DataQualitySchema(AutoSchemaHelper):
-    def __init__(self):
-        super().__init__()
+    def __init__(self, *args):
+        super().__init__(*args)
 
         self.manual_fields = {
             ('POST', 'create'): [
                 self.org_id_field(),
-                self.form_field(
+                self.body_field(
                     name='data_quality_ids',
                     required=True,
-                    description="An object containing IDs of the records to perform data quality checks on. Should contain two keys- property_state_ids and taxlot_state_ids, each of which is an array of appropriate IDs."
+                    description="An object containing IDs of the records to perform data quality checks on. Should contain two keys- property_state_ids and taxlot_state_ids, each of which is an array of appropriate IDs.",
+                    params_to_formats={
```

Comment:
This provides a nice "default value" on the Swagger pager, but I was struggling to find a pattern for us developers to use and reuse.

<img width="837" alt="Screen Shot 2020-04-23 at 2 26 36 PM" src="https://user-images.githubusercontent.com/30608004/80146544-3da04880-856f-11ea-8497-afbed87abe66.png">